src/components/PostList/PostList.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo/PostInfo';
+
+type Props = {
+  content: Post[];
+};
+
+export const PostList: React.FC<Props> = props => {
+  const { content } = props;
+
+  return (
+    <ul>
+      {content.map((post: Post) => (
```

Comment:
```suggestion
      {posts.map((post: Post) => (
```
content це дуже абстрактна назва